src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastDetails", () => {
+  const validProps = {
+    date: 1111111,
+    humidity: 60,
+    temperature: {
+      max: 22,
+      min: 12,
+    },
+    wind: {
+      speed: 80,
+      direction: "e",
+    },
+  };
+
+  it("renders correctly", () => {
```

Comment:
Could some more tests be added here to validate if the values being passed in are being displayed as they should be?